jargons/angularjs.md

```diff
@@ -0,0 +1,10 @@
+---
+title: angularjs
+sameWith:
+- angular
+tags:
+- javascript
+- mvvm
+---
+
+Popüler bir arayüz framework'üdür. **İki yönlü veri bağlama** yöntemi ile çalışır. Böylece veri katmanı ve görsel katman birbirini sürekli olarak güncellerler. Bu yöntemin bir diğer adı [MVVM](/mvvm)'dir.
```

Comment:
@f Her ne kadar sevmesem de `framework` yerine `çatı` kullanmak daha dogru olur mu acaba?

Reply:
Ya da framework'ü linkleyip bu sekilde birakabiliriz. Ekstradan da framework için bir tanimlama yapilabilir sonra.